lib/rucio/tests/common.py

```diff
@@ -36,6 +37,31 @@
                                       reason="specific belleii tests")
 
 
+def is_influxdb_available():
+    """Return True if influxdb is available, else return False."""
+    try:
+        response = requests.get('http://localhost:8086/ping')
+        if response.status_code == 204:
+            return True
+    except requests.exceptions.ConnectionError:
+        print('InfluxDB is not running at localhost:8086')
+        return False
+
+
+def is_elasticsearch_available():
+    """Return True if elasticsearch is available, else return False."""
+    try:
+        response = requests.get('http://localhost:9200/')
+        if response.status_code == 200:
+            return True
+    except requests.exceptions.ConnectionError:
+        print('Elasticsearch is not running at localhost:9200')
+        return False
+
+
+skip_missing_elasticsearch_influxdb_in_env = pytest.mark.skipif(not (is_influxdb_available() and is_elasticsearch_available()), reason='influxdb is not available')
```

Comment:
@cserf you can use this as a decorator, to mark tests that should run only when elastic and influxdb are present in the test environment ;)